### PR TITLE
Income and Assets (21P-0969) endpoint boilerplate

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Summary

Adds boilerplate code for the income and assets (`/v0/form0969`) endpoint. Submissions are successfully posted to the v0/form0969 endpoint, but aren't yet forwarded to VBA. PDF mapping, VBA submission, and logging/metrics are planned in later issues in the epic.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82554

## Testing done

- [x] Added unit tests for new route
- [x] Ran Swagger UI locally to confirm presence of endpoint (following guidelines in `app/swagger/readme.md`)
- [x] [Updated vets-website](https://github.com/department-of-veterans-affairs/vets-website/pull/30053) to allow submission to the endpoint locally, and confirmed successful submission.

## Screenshots
Screenshot 2024-05-29 at 1 47 39 PM
Screenshot 2024-05-29 at 1 49 32 PM

## What areas of the site does it impact?
Income and Assets

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

### CODE REVIEW

**Comment by TaiWilkin** [comment]
Before this can be merged, [the related schema update](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/899/) must be merged, and the `vets-json-schema` gem must be updated.

File: spec/requests/swagger_spec.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@
     it 'has valid json' do
       get '/v0/apidocs.json'
       json = response.body
+      print JSON.parse(json)
```
[the remark this answers is not in the document]
Comment (TaiWilkin) [reply spec/requests/swagger_spec.rb]: Thanks for catching that!

File: config/routes.rb [hunk 2]
```diff
@@ -141,6 +141,9 @@
       resources :burial_claims, only: %i[create show]
     end
 
+    post 'form0969', to: 'income_and_assets_claims#create'
+    get 'form0969', to: 'income_and_assets_claims#show'
+
```
Comment (wayne-weibel) [inline config/routes.rb:146]: can wrap in flipper enabled check to be sure nothing can hit the endpoint
```
    if Flipper.enabled?(:income_and_assets_form_enabled)
      post 'form0969', to: 'income_and_assets_claims#create'
      get 'form0969', to: 'income_and_assets_claims#show'
    end
```

**Review by wayne-weibel — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 2 above)
Comment (TaiWilkin) [reply config/routes.rb:146]: Reversed the PR making this change, because the Flipper addition to routes.rb caused CI issues

**Review by wayne-weibel — APPROVED** [review]

**Review by ericboehs — APPROVED** [review]

**Pull request merged**